src/components/login/register.tsx

```diff
@@ -1,42 +1,88 @@
-import { Button, Classes, InputGroup } from "@blueprintjs/core";
+import {
```

Comment:
Also similar question from my comment in navigation.tsx, should we make this route not even available to people if they aren't admins? i.e. what if they just enter this url directly?